### DISCUSSION TITLE
Why is code property of error object undefined?

### DISCUSSION THREAD

**Discussion by Diwannist** [supabase/supabase#28025] [Questions] [answered]
I'm using JavaScript and I want to access the code property of the error object when using Supabase authentication. I can successfully access the name property, status property, and message property but not the code property.

This is what Supabase docs mention:

> Errors originating from the server API classed as AuthApiError always have a code property that can be used to identify the error returned by the server.

source: https://supabase.com/docs/reference/javascript/auth-error-codes

Yet I am consistently getting "undefined" when accessing the code property. For example, I wrote some code to output all properties of the error object to get some clarification and this is what I received:

> message: Invalid login credentials
__isAuthError: true
name: AuthApiError
status: 400
code: undefined
line: 7
column: 4034
sourceURL: https://cdn.jsdelivr.net/npm/@supabase/supabase-js@2
stack: w@https://cdn.jsdelivr.net/npm/@supabase/supabase-js@2:7:4034
T@https://cdn.jsdelivr.net/npm/@supabase/supabase-js@2:7:4221
@https://cdn.jsdelivr.net/npm/@supabase/supabase-js@2:7:6534

**Comment by silentworks** [comment, marked as answer]
It seems there was no error code assigned to this as of yet. I've raised this with the Supabase auth team as I got the same results when testing signing in with email/password too.

Edit: The Supabase auth team is aware of this issue and is tracking it on the Supabase Auth GitHub repo https://github.com/supabase/auth/issues/1631
